src/test/java/com/wikia/webdriver/testcases/articlecrudtests/ArticleActionsAdminTests.java

```diff
@@ -30,14 +36,19 @@ public void deleteUndeleteArticle() {
     ArticlePageObject article = new ArticlePageObject().open(articleTitle);
     DeletePageObject deletePage = article.deleteUsingDropdown();
     deletePage.submitDeletion();
-    SpecialRestorePageObject restore =
-        article.getBannerNotifications().clickUndeleteLinkInBannerNotification();
+    List<Notification> confirmNotifications = article.getNotifications(NotificationType.CONFIRM);
+    Assertion.assertEquals(confirmNotifications.size(),1,
+            "Number of action confirming notifications is invalid");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Number of action confirming notifications is invalid" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)